zombienet.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -Eex
+
+case "$(uname -s)" in
+  Linux*)   machine=linux;;
+  Darwin*)  machine=macos;;
+  *)        exit 1;;
+esac
+
+provider=${ZOMBIENET_PROVIDER:-native}
+version=${ZOMBIENET_VERSION:-v1.2.59}
+runtime=${RUNTIME:-t3rn-parachain}
+pdot_branch=${PDOT_BRANCH:-release-v0.9.27}
+root_dir=$(git rev-parse --show-toplevel)
+
+mkdir -p /tmp/zombienet/bin
+
+if [[ ! -x /usr/local/bin/zombienet ]]; then
+  curl -fL# -o /usr/local/bin/zombienet https://github.com/paritytech/zombienet/releases/download/$version/zombienet-$machine
+  chmod +x /usr/local/bin/zombienet
+fi
+
+if [[ ! -x /tmp/zombienet/bin/polkadot ]]; then
```

Comment:
we should just pull the binaries into .bin and ignore them, so we don't have to pull polkadot/polkadot-parachain every time